activerecord/lib/active_record/connection_adapters/abstract/schema_definitions.rb

```diff
@@ -328,6 +328,237 @@ def primary_key(name, type = :primary_key, **options)
+      # :call-seq: float(*names, **options)
+      #
+      # Adds a +float+ column for storing approximate decimal numbers.
+      #
+      # See {connection.add_column}[rdoc-ref:ConnectionAdapters::SchemaStatements#add_column]
+      # for available options.
+      #
+      # Example:
+      #
+      #   create_table :reviews do |t|
+      #     t.float :rating
+      #   end
+
+      ##
+      # :method: integer
+      # :call-seq: integer(*names, **options)
+      #
+      # Adds an +integer+ column for storing whole numbers.
+      #
+      # See {connection.add_column}[rdoc-ref:ConnectionAdapters::SchemaStatements#add_column]
+      # for available options.
+      #
+      # Example:
+      #
+      #   create_table :books do |t|
+      #     t.integer :ratings, array: true, default: []
+      #   end
+
+      ##
+      # :method: json
+      # :call-seq: json(*names, **options)
+      #
+      # Adds a +json+ column for storing structured data in JSON format.
+      #
+      # Use this for unindexed, flexible key-value data. For indexed JSON support, prefer +jsonb+ on PostgreSQL.
+      #
+      # Example:
+      #
+      #   create_table :accounts do |t|
+      #     t.json :settings
+      #   end
+
+      ##
+      # :method: string
+      # :call-seq: string(*names, **options)
+      #
+      # Adds a +string+ column for storing short text values.
+      #
+      # See {connection.add_column}[rdoc-ref:ConnectionAdapters::SchemaStatements#add_column]
+      # for available options.
+      #
+      # You can limit the maximum length using the +:limit+ option.
+      #
+      # Example:
+      #
+      #   create_table :labels do |t|
+      #     t.string :name, limit: 50
+      #   end
+
+      ##
+      # :method: text
+      # :call-seq: text(*names, **options)
+      #
+      # Adds a +text+ column for storing longer, unbounded text data.
+      #
+      # See {connection.add_column}[rdoc-ref:ConnectionAdapters::SchemaStatements#add_column]
+      # for available options.
+      #
+      # Example:
+      #
+      #   create_table :reviews do |t|
+      #     t.text :description
+      #   end
+
+      ##
+      # :method: time
+      # :call-seq: time(*names, **options)
+      #
+      # Adds a +time+ column for storing time-of-day values (without date).
+      #
+      # See {connection.add_column}[rdoc-ref:ConnectionAdapters::SchemaStatements#add_column]
+      # for available options.
+      #
+      # Example:
+      #
+      #   create_table :lessons do |t|
+      #      t.time :sunrise
+      #   end
+
+      ##
+      # :method: timestamp
+      # :call-seq: timestamp(*names, **options)
+      #
+      # Adds a +timestamp+ column for storing date and time values.
+      #
+      # This is similar to +datetime+ and may be treated interchangeably on some databases.
+      #
+      # See {connection.add_column}[rdoc-ref:ConnectionAdapters::SchemaStatements#add_column]
+      # for available options.
+      # Example:
```

Comment:
```suggestion
      #
      # Example:
```

Reply:
A hard refresh of the browser seems to be showing it correctly now.